Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -296,9 +302,18 @@ void VideoConfig::Save(const std::string& ini_file)
   settings->Set("OverlayStats", bOverlayStats);
   settings->Set("OverlayProjStats", bOverlayProjStats);
   settings->Set("DumpTextures", bDumpTextures);
-  settings->Set("HiresTextures", bHiresTextures);
-  settings->Set("ConvertHiresTextures", bConvertHiresTextures);
-  settings->Set("CacheHiresTextures", bCacheHiresTextures);
+  if (!settings->Get("CustomTextures", bCustomTextures))
+      settings->Set("HiresTextures", bCustomTextures);
+  else
+      settings->Set("CustomTextures", bCustomTextures);
+  if (!settings->Get("ConvertCustomTextures", bConvertCustomTextures))
+      settings->Set("HiresCustomTextures", bConvertCustomTextures);
+  else
+      settings->Set("ConvertCustomTextures", bConvertCustomTextures);
+  if (!settings->Get("CacheCustomTextures", bCacheCustomTextures))
+      settings->Set("CacheHiresTextures", bCacheCustomTextures);
+  else
+      settings->Set("CacheCustomTextures", bCacheCustomTextures);
```

Comment:
Having fallbacks like this in `Save` doesn't make much sense. If you're changing the config name, Dolphin is supposed to save under the new name, otherwise the new config name won't ever be used unless the user manually sets it in their INI.